src/__Private/CLIBase.php

```diff
@@ -206,6 +208,48 @@ public function __construct(
     exit(1);
   }
 
+  private static function extractOptions(string $arg, CLIOptions\CLIOptionType $type): dict<string, ?string> {
+    $options = vec[];
+
+    switch ($type) {
+      case CLIOptions\CLIOptionType::LONG:
+        $options[] = $arg;
+        break;
+
+      case CLIOptions\CLIOptionType::SHORT:
+        $arg_length = Str\length($arg);
+        for ($index = 0; $index < $arg_length; $index++) {
+          $options[] = $arg[$index];
+        }
+        break;
+
+      default:
```

Comment:
Can remove this, Hack can do exhaustiveness checking on enums.

Reply:
Actually, I omitted one enum value so could not remove default branch